p8_integration_tests/scripts_test/build_script.py

```diff
@@ -67,10 +70,14 @@ def add_scripts(a_dir, prefix_len, test_file, exceptions, broken):
     examples_script = os.path.join(tests_dir, "examples_auto_test.py")
     examples_header = os.path.join(tests_dir, "examples_header.py")
     copyfile(examples_header, examples_script)
+    exceptions = ["pushbot_ethernet_example.py"]
+    if len(sys.argv) > 1:  # 1 is the script name
```

Comment:
Not sure I get this; do you mean skip if we add more arguments than just the file name?  I am guessing this will therefore skip if you use py.test?

Reply:
This code is intended to be run from the Jenkinsfile with the command
 sh 'python sPyNNaker8/p8_integration_tests/scripts_test/build_script.py'

I needed a way to flag between running build_script from the cron and none crom jenkins files so I could exclude the known long running ones from the none cron. The easiest was a boolean test is there or is there not an extra parameter.  So in the none cron the call is
 sh 'python sPyNNaker8/p8_integration_tests/scripts_test/build_script.py short'

Although it could just as easily be:
 sh 'python sPyNNaker8/p8_integration_tests/scripts_test/build_script.py bacon'

Reply:
OK